src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1205,109 +1204,237 @@ export default class ChargingStationService {
-        filteredRequest = ChargingStationValidator.getInstance().validateChargingStationActionResetReq(req.body);
-        result = await ChargingStationService.executeChargingStationCommand(
-          req.tenant, req.user, chargingStation, action, command, filteredRequest.args);
-        break;
-      // Other commands
-      default:
-        // Log Specific Schema has not been verified:
-        await Logging.logError({
-          tenantID: req.tenant.id,
-          siteID: chargingStation.siteID,
-          siteAreaID: chargingStation.siteAreaID,
-          companyID: chargingStation.companyID,
-          chargingStationID: chargingStation.id,
-          source: chargingStation.id,
-          user: req.user,
-          action: action,
-          module: MODULE_NAME, method: 'handleAction',
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          message: `Command '${command}' has not its own validation schema.`
-        });
-        throw new AppError({
-          source: Constants.CENTRAL_SERVER,
-          errorCode: HTTPError.GENERAL_ERROR,
-          message: `Command '${command}' has not its own validation schema.`,
-          module: MODULE_NAME,
-          method: 'handleAction'
-        });
+    // Get the OCPP Client
+    const chargingStationClient = await ChargingStationClientFactory.getChargingStationClient(req.tenant, chargingStation);
+    if (!chargingStationClient) {
+      throw new BackendError({
+        source: chargingStation.id,
+        action: action,
+        module: MODULE_NAME, method: 'handleChargingStationCommand',
+        message: 'Charging Station is not connected to the backend',
+      });
+    }
+    let filteredRequest: any;
+    try {
+      let result: any;
+      switch (command) {
+        // Clear Cache
+        case Command.CLEAR_CACHE:
+          filteredRequest = ChargingStationValidator.getInstance().validateChargingStationActionCacheClearReq(req.body);
+          result = result = await chargingStationClient.clearCache();
+          break;
+        // Change Availability
+        case Command.CHANGE_AVAILABILITY:
+          filteredRequest = ChargingStationValidator.getInstance().validateChargingStationActionAvailabilityChangeReq(req.body);
+          result = await chargingStationClient.changeAvailability({
+            connectorId: filteredRequest.args.connectorId,
+            type: filteredRequest.args.type
+          });
+          break;
+        case Command.GET_CONFIGURATION:
+          filteredRequest = ChargingStationValidator.getInstance().validateChargingStationActionConfigurationGetReq(req.body);
+          result = await chargingStationClient.getConfiguration({ key: filteredRequest.args.key });
+          break;
+        // Change Configuration
+        case Command.CHANGE_CONFIGURATION:
+          filteredRequest = ChargingStationValidator.getInstance().validateChargingStationActionConfigurationChangeReq(req.body);
```

Comment:
Create a method like executeChargingStationChangeConfiguration() so the switch stays readable